frontend/app/tab/tab.scss

```
@@ -1,6 +1,28 @@
+// Uses CSS @property for an animatable custom property (Chromium 78+).
+@property --breathe-phase {
+    syntax: "<number>";
+    initial-value: 0;
+    inherits: true;
+}
+
+:root {
+    animation: indicatorBreathePhase 16s ease-in-out infinite;
+}
+
+@keyframes indicatorBreathePhase {
```

Stylelint (17.2.0)

[error] Expected keyframe name "indicatorBreathePhase" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)